metadata.yaml

```diff
@@ -26,11 +26,11 @@ requires:
     interface: prometheus_remote_write
   metrics-endpoint:
     interface: prometheus_scrape
-  logging:
+  logging-consumer:
     interface: loki_push_api
 
 provides:
-  log_proxy:
+  logging-provider:
```

Comment:
Can this be made less ambiguous using other names? I'm thinking "write-logs" and "read-logs" or similar. It adds no additional value to use the exact term used as the parent key name.